dev-util/premake/premake-5.0.0_alpha16.ebuild

```
@@ -0,0 +1,32 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_P=${P/_/-}
+MY_PV=${PV/_/-}
+
+DESCRIPTION="A makefile generation tool"
+HOMEPAGE="https://premake.github.io/"
+SRC_URI="https://github.com/${PN}/${PN}-core/archive/v${MY_PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="BSD"
+
+SLOT=$(ver_cut 1)
+
+KEYWORDS="~amd64 ~ppc ~x86"
+
+S="${WORKDIR}/${PN}-core-${MY_PV}"
```

Could use some re-ordering to match skel.ebuild and less spacing, and I'd argue this ver_cut isn't worth.. full SLOT aren't auto-changed on a whim and better to make it clear.
```suggestion
S="${WORKDIR}/${PN}-core-${MY_PV}"

LICENSE="BSD"
SLOT="5"
KEYWORDS="~amd64 ~ppc ~x86"
```